policies/taskrun.rego

```diff
@@ -0,0 +1,33 @@
+package hacbs.contract.taskrun
+
+validAnnotations := ["Red Hat", null]
+validRegistries := ["docker.io", "registry.access.redhat.com"]
```

Comment:
An idea: Could put these lists in policies/lib/config.rego .

There are pros and cons, but something to consider.

Reply:
I will look at that. I need to rework the tests also.